### ISSUE TITLE
Low-grade vesicoureteral reflux

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6823]
**Preferred term label:**
Low-grade vesicoureteral reflux

**Synonyms**

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux without dilation of the renal calyces (Grade I-II)
PMID: 18322164

**Parent term (use hpo.jax.org/app)**
Vesicoureteral reflux HP:0000076

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
